File: VB/WindowsApplication3/MVVM/Views/CustomersView.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Windows.Forms

Namespace MVVMSample
    Partial Public Class CustomersView
        Inherits UserControl

        Public Sub New()
            InitializeComponent()
            If Not mvvmContext.IsDesignMode Then
                InitializeBindings()
            End If
        End Sub
        Private Sub InitializeBindings()
            mvvmContext.ViewModelType = GetType(CustomersViewModel)
            Dim fluent = mvvmContext.OfType(Of CustomersViewModel)()
            fluent.SetBinding(gridControl1, Function(c) c.DataSource, Function(x) x.Entities)
            fluent.BindCommand(clearNavButton, Sub(x) x.Clear())
            fluent.BindCommand(removeLastNavButton, Sub(x) x.RemoveLast())
            fluent.BindCommand(addNavButton, Sub(x) x.Add())
        End Sub
    End Class
End Namespace

File: CS/WindowsApplication3/MVVM/Views/CustomersView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;

namespace MVVMSample {
    public partial class CustomersView : UserControl {
        public CustomersView() {
            InitializeComponent();
            if (!mvvmContext.IsDesignMode) {
                InitializeBindings();
            }
        }
        private void InitializeBindings() {
            mvvmContext.ViewModelType = typeof(CustomersViewModel);
            var fluent = mvvmContext.OfType<CustomersViewModel>();
            fluent.SetBinding(gridControl1, c => c.DataSource, x => x.Entities);
            fluent.BindCommand(clearNavButton, x => x.Clear());
            fluent.BindCommand(removeLastNavButton, x => x.RemoveLast());
            fluent.BindCommand(addNavButton, x => x.Add());
        }
    }
}
